main.py

```diff
@@ -0,0 +1,51 @@
+import time
+from control import XboxController, OperatingMode, handle_operating_mode, Wheels, SoundEffects
+import vlc
+
+def main():
+
+    max_angular_velocity = 200
+    max_linear_velocity = 500
+
+    # Instantiate the Components
+    # wheels = Wheels(
+    #     max_linear_velocity=max_linear_velocity,
+    #     max_angular_velocity=max_angular_velocity,
+    # )
+
+    joy = XboxController()
+    sounds_effects = SoundEffects()
+    operating_mode = OperatingMode.STATIONARY
+
+    while True:
+
+        control_inputs = joy.read()
+
+        new_operating_mode = handle_operating_mode(control_inputs["operating_mode"])
+        
+        if new_operating_mode and operating_mode != new_operating_mode:
+            operating_mode = new_operating_mode
+            sounds_effects.play_change_mode()
+        
+        if operating_mode == OperatingMode.DRIVE:
+            # Use Controller to drive the rover and control the mast
+            print("DRIVE")
+            # wheels.handle_inputs(control_inputs["wheels"])
+        elif operating_mode == OperatingMode.ROBOTIC_ARM:
```

Comment:
Could we have the arm control on another thread (i.e., background loop)? Since we don't have to explicitly control the arms, and it just needs to follow the predetermined paths, I think there is no need to go into this mode.

Reply:
On a similar note, I think the face following can also be put on a thread that runs in the background, following faces regardless of the operating modes.

Reply:
Yeah exactly, so in this conditional part, we can check whether the background thread has started. If it has started then we continue. If it has not started then we start the background thread. Will need to handle killing the thread when changing operating modes